wrapplog.py

```diff
@@ -1,22 +1,17 @@
 import os
-import json
 import inspect
 import collections
 from datetime import datetime
 
-import structlog
 from structlog.processors import JSONRenderer
 from structlog import wrap_logger, PrintLogger, PrintLoggerFactory
 
-
-# Deprecated
-def start_logging(output=None):
```

Comment:
Have you verified that we're safe to remove it? Or is it a way to enforce the removal? :)